frontend/src/app/constants.ts

```diff
@@ -43,7 +43,7 @@ export const PARAMETERS: { [key: string]: string } = {
   sampler: 'Sampler',
   seed: 'Seed',
   img2imgStrength: 'img2img Strength',
-  gfpganStrength: 'GFPGAN Strength',
+  facetoolStrength: 'GFPGAN Strength',
```

Comment:
Should rename to Facetool Strength

Reply:
Updated with fix for this and a few other related places where things were named incorrectly.